openapi.yml

```diff
@@ -62,6 +62,72 @@ paths:
           description: FlowMod messages sent.
         '400':
            description: Invalid JSON flow.
+  '/api/kytos/flow_manager/v2/flows_by_switch':
+    post:
+      tags:
+        - Add
+      summary: Insert flows by switch.
+      parameters:
+        - name: force
+          in: query
+          schema:
+            type: boolean
+          description: Option to ignore switch connection errors.
+          required: false
+          example: "true"
+      requestBody:
+        description: Flows by the especified switch.
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              patternProperties:
```

Comment:
On https://editor.swagger.io/, it's pointing out some errors, can you review the errors? Ignore the one about POST with body deletion though:

![20241106_130903](https://github.com/user-attachments/assets/cdaf9319-c47b-4246-907a-6c8c482bc4d6)

Reply:
Oh ye, that editor does not support openapi 3.1.0. The new one does, https://editor-next.swagger.io/